test/blackbox-tests/test-cases/configurator-import.t/configure/configure.ml

```diff
@@ -0,0 +1,93 @@
+(*
+ * OWL - OCaml Scientific and Engineering Computing
+ * Copyright (c) 2016-2022 Liang Wang <liang@ocaml.xyz>
+ *)
+
+module C = Configurator.V1
+
+let detect_system_header =
+  {|
+  #if __APPLE__
+    #include <TargetConditionals.h>
+    #if TARGET_OS_IPHONE
+      #define PLATFORM_NAME "ios"
+    #else
+      #define PLATFORM_NAME "mac"
+    #endif
+  #elif __linux__
+    #if __ANDROID__
+      #define PLATFORM_NAME "android"
+    #else
+      #define PLATFORM_NAME "linux"
+    #endif
+  #elif WIN32
+    #define PLATFORM_NAME "windows"
+  #endif
```

Comment:
If the test will only run on windows, I think all these ifdef's are not necessary.

Reply:
This is broken on windows, but if I add the test we may as well run it everywhere. I can simplify it with a simpler file to include, if you prefer. It does not really matter in the end what I define in there

Reply:
> This is broken on windows, but if I add the test we may as well run it everywhere. I can simplify it with a simpler file to include, if you prefer. It does not really matter in the end what I define in there

Sure, we can run it everywhere if the output is the same everywhere.

Reply:
Yes. The output, in all the three commits, was independent of the platform